testing/SendLog/SendLog.py

```diff
@@ -0,0 +1,46 @@
+## Hrishi Hiraskar
+## 31 August 2016
+
+import gevent
+import time
+from gevent import monkey
+from gevent.pywsgi import WSGIServer
+from flask import Flask, request, Response, render_template
+
+monkey.patch_all()
+
+app = Flask(__name__)
+
+def event_stream():
+	# Open the log file
+	log_dir = "../../bin/"				# Log file directory
+	log_name = "scilab-log-0.txt"		# Log file name
+	logfile = open(log_dir + log_name, "r")
+	# Seek file pointer to the end of file
+	logfile.seek(0,2)
+	# Start sending log
+	LOOK_DELAY = 0.01					# Delay time to look for new line (in s)
+	while 1:
```

Comment:
When does the loop look end?

The first set of few lines are for initialization.
And after that, the points start.
The last set of few lines are for ending.

Ideally what you should do, is maintain a list with the set of Initialization id's.

You will have to parse the incoming line to do this. So there would be three cases: Initialization, Point Data, Ending. 

Once, an item in the initialization list has ended, they should be removed from the list. 

When the list is empty, the program should stop running.